lib/api/1.1/southbound/profiles.js

```diff
@@ -31,9 +31,13 @@ function profilesRouterFactory (
      */
 
     router.get('/profiles', function (req, res) {
+        if (req.query.mac && req.query.ip) {
```

Comment:
Could you help move the description like 'Creates lookup entry based on DHCP bootfile option request query parameters' to the annotations here? the message in PR  wouldn't be included in codes, and annotations here will help a lot for code readablility